packages/istanbul-api/test/reporter.test.js

```diff
@@ -0,0 +1,106 @@
+/* globals context, describe, it */
+
+var assert = require('chai').assert,
+    configuration = require('../lib/config'),
+    coverage = require('../../istanbul-lib-coverage'),
```

Comment:
Sorry I didn't catch this previously, could you change this to simply `coverage = require('istanbul-lib-coverage')`?  This way the module is imported from `node_modules` instead of directly from the relative directory.

Reply:
Done!